src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource_ProduceTags.cs

```diff
@@ -258,13 +255,22 @@ private async Task RecomputeTagsAsync(bool highPriority, CancellationToken cance
 
                     this.CachedTagTrees = newTagTrees;
                     this.State = context.State;
+                    if (this._subjectBuffer.CurrentSnapshot.Version.VersionNumber == subjectBufferVersion)
+                    {
+                        // Only clear the accumulated text changes if the subject buffer didn't change during the
+                        // tagging operation. Otherwise, it is impossible to know which changes occurred prior to the
+                        // request to tag, and which ones occurred during the tagging itself.
+                        this.AccumulatedTextChanges = null;
+                    }
```

Comment:
ok.  this change seems very safe to me.  That said, i would prefer the comment on it (or some documentation somewhere) be expanded a bit.  namely, stating that it's *ok* to not clear this out, as the only impact is slightly worse performance for some taggers by indicating potentially *more* of the document was changed than actually changed.

Basically, i'd like docs somewhere that this is more a conservative estimate of what changed, and it is ok to be more (but not less) than what actually did change.  Thanks!